appveyor.yml

```diff
@@ -23,20 +23,34 @@ environment:
       EXAMPLE: 01-executable
       CONFIG: Release
 
+    - TOOLCHAIN: "vs-14-2015"
+      EXAMPLE: 02-library
+      CONFIG: Release
+
+    - TOOLCHAIN: "vs-14-2015"
+      EXAMPLE: 03-shared-link
+      CONFIG: Release
+
     - TOOLCHAIN: "mingw"
       EXAMPLE: 01-executable
       CONFIG: Release
 
-    - TOOLCHAIN: "msys"
+    - TOOLCHAIN: "mingw-cxx14"
       EXAMPLE: 01-executable
       CONFIG: Release
+      CMAKE_MAKE_PROGRAM: mingw32-make.exe
```

Comment:
Do we really need this?

Reply:
Not including this results in: 

-- [polly] Used toolchain: mingw / gcc / c++17 support
-- [polly *** DEBUG ***] Used tag: mingw-cxx17
-- [polly *** DEBUG ***] CMAKE_DEBUG_POSTFIX is empty
-- [polly *** DEBUG ***] CMAKE_DEBUG_POSTFIX set to 'd'

gcc not found

https://ci.appveyor.com/project/ckaminski/polly/build/job/wie2idct71mnpsop